.github/workflows/continuous-integration-stager.yml

```diff
@@ -35,9 +35,6 @@ jobs:
     permissions:
       # required for all workflows
       security-events: write
-      # only required for workflows in private repositories
-      actions: read
-      contents: read
```

Comment:
Because it is redundant with https://github.com/kokkos/kokkos/blob/69911b3e0ed2cb3bb9cb2682360c6a57723dbbea/.github/workflows/continuous-integration-stager.yml#L16 or based on the comment?